src/main/java/uk/gov/justice/hmpps/prison/service/xtag/XtagContent.java

```diff
@@ -113,4 +113,100 @@ public class XtagContent {
+        if (p_old_alert_time != null) sb.append(", p_old_alert_time=").append(p_old_alert_time);
+        if (p_alert_date != null) sb.append(", p_alert_date=").append(p_alert_date);
+        if (p_alert_time != null) sb.append(", p_alert_time=").append(p_alert_time);
+        if (p_expiry_date != null) sb.append(", p_expiry_date=").append(p_expiry_date);
+        if (p_expiry_time != null) sb.append(", p_expiry_time=").append(p_expiry_time);
+        if (p_movement_date != null) sb.append(", p_movement_date=").append(p_movement_date);
+        if (p_movement_time != null) sb.append(", p_movement_time=").append(p_movement_time);
+        if (p_incident_case_id != null) sb.append(", p_incident_case_id=").append(p_incident_case_id);
+        if (p_party_seq != null) sb.append(", p_party_seq=").append(p_party_seq);
+        if (p_requirement_seq != null) sb.append(", p_requirement_seq=").append(p_requirement_seq);
+        if (p_question_seq != null) sb.append(", p_question_seq=").append(p_question_seq);
+        if (p_response_seq != null) sb.append(", p_response_seq=").append(p_response_seq);
+        if (p_table_name != null) sb.append(", p_table_name=").append(p_table_name);
+        if (p_alert_type != null) sb.append(", p_alert_type=").append(p_alert_type);
+        if (p_movement_type != null) sb.append(", p_movement_type=").append(p_movement_type);
+        if (p_identifier_type != null) sb.append(", p_identifier_type=").append(p_identifier_type);
+        if (p_bed_assign_seq != null) sb.append(", p_bed_assign_seq=").append(p_bed_assign_seq);
+        if (p_living_unit_id != null) sb.append(", p_living_unit_id=").append(p_living_unit_id);
+        if (p_new_record != null) sb.append(", p_new_record=").append(p_new_record);
+        if (p_record_deleted != null) sb.append(", p_record_deleted=").append(p_record_deleted);
+        if (p_timestamp != null) sb.append(", p_timestamp=").append(p_timestamp);
+        if (p_event_id != null) sb.append(", p_event_id=").append(p_event_id);
+        if (p_start_time != null) sb.append(", p_start_time=").append(p_start_time);
+        if (p_end_time != null) sb.append(", p_end_time=").append(p_end_time);
+        if (p_event_class != null) sb.append(", p_event_class=").append(p_event_class);
+        if (p_event_type != null) sb.append(", p_event_type=").append(p_event_type);
+        if (p_event_sub_type != null) sb.append(", p_event_sub_type=").append(p_event_sub_type);
+        if (p_event_status != null) sb.append(", p_event_status=").append(p_event_status);
+        if (p_agy_loc_id != null) sb.append(", p_agy_loc_id=").append(p_agy_loc_id);
+        if (p_iep_level_seq != null) sb.append(", p_iep_level_seq=").append(p_iep_level_seq);
+        if (p_iep_level != null) sb.append(", p_iep_level=").append(p_iep_level);
+        if (p_audit_module_name != null) sb.append(", p_audit_module_name=").append(p_audit_module_name);
+        return "{" + chop(sb.toString()) + "}";
+    }
```

Comment:
This removes large amounts of redundant output but more importantly matches the output from the new event service for easier parallel running comparison